docs/content/docs/community/how-it-works.md

```diff
@@ -61,6 +61,9 @@ an ACK service controller will look for any drift and attempt to remediate. As
 part of the remediation, an ACK service controller will reconfigure the managed
 resource based on the `Spec`.
 
+For more information about configuring the drift remediation period, see
```

Comment:
Can you please modify the sentence above to read `...every 10 hours (unless a different frequency is configured)...`?  It's great to point the user to more info, but folks who read only this page might come away with the impression that it's always 10 hours.